createdisk.sh

```diff
@@ -81,6 +81,9 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl rmp $(sudo crictl
 # Remove pull secret from the VM
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -f /var/lib/kubelet/config.json'
 
+# Remove audit logs
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -f /var/log/kube-apiserver/*'
```

Comment:
@guillaumerose what if we just delete everything from `/var/log` does those directory needs to be present for some container mount?

Reply:
good point, will try!